core/decs.h

```diff
@@ -196,9 +196,11 @@
 
 // EOS
 #define EOS_TYPE_GAMMA (0)
+#define EOS_TYPE_GAMMA_GASPRESS (0)
+#define EOS_TYPE_GAMMA_RADPRESS (0)
```

Comment:
These all can't be zero. That means `EOS_TYPE_GAMMA_GASPRESS` and `EOS_TYPEGAMMA_RADPRESS` will resolve to the same number. That's probably not your intent. Each EOS type needs to resolve to a different number in the macro.